docs/detections/api/rules/rules-api-import.asciidoc

```diff
@@ -8,6 +8,8 @@ Imports rules from an `.ndjson` file. The following configuration items are also
 
 NOTE: Console supports {es} APIs only. Console doesn't allow interactions with {kib} APIs. You must use `curl` or another HTTP tool instead. For more information, refer to {kibana-ref}/console-kibana.html[Run {es} API requests]. Please also note this API supports {kibana-ref}/api.html#token-api-authentication[Token-based authentication] only.
 
+NOTE: You need at least `Read` privileges for the `Action and Connectors` feature to import rules with actions. If you are importing rules without actions, no `Action and Connectors` feature privileges are required. Refer to <<enable-detections-ui>> for more information.
```

Comment:
First edit just for voice, but for the second edit I think it's clearer to say something is "not required" vs. separating "no" and "required" with a long phrase.
```suggestion
NOTE: You need at least `Read` privileges for the `Action and Connectors` feature to import rules with actions. If you're importing rules without actions, `Action and Connectors` feature privileges are not required. Refer to <<enable-detections-ui>> for more information.
```